src/test/java/com/commercetools/sync/products/utils/ProductSyncUtilsTest.java

```diff
@@ -230,4 +237,43 @@ void buildActions_FromDraftsWithSameNameValues_ShouldNotBuildUpdateActions() {
 
         assertThat(updateActions).isEmpty();
     }
+
+    @Test
+    void buildActions_FromDraftsWithSameForAllAttribute_ShouldBuildUpdateActions() {
```

Comment:
It's better if we can have a test case which has no SameForAll attributes. Just a notify that we don't reduce our code coverage 😄

Reply:
I think the rest of the test cases are not having sameForAll attributes.
Or did i miss your point?

Reply:
Yeah, i agree this code coverage was driving me crazy.
I added one more test "buildActions_FromDraftsWithDifferentAttributes_ShouldBuildUpdateActions()" for that in order to make sure other attribute updates can be duplicated unlike the sameForAll attributes.